WcaOnRails/app/webpacker/components/StaticPages/About.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { List } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import I18nHTMLTranslate from '../I18nHTMLTranslate';
+
+const DONATE_PAYPAL_URL = 'https://www.paypal.com/donate/?hosted_button_id=W5JL8J4S8UTJU';
+const DONATE_GUIDESTAR_URL = 'https://www.nfggive.com/guidestar/82-3825954';
+
+/**
+ * @param {{ name: string, country_name: string, id: number, [k: *]: * }[]} currentBoardMembers
+ * @returns {JSX.Element}
+ * @constructor
+ */
+function About({ currentBoardMembers }) {
+  return (
+    <React.StrictMode>
+      <div className="jumbotron">
+        <p className="lead">
+          {I18n.t('about.donation_banner.content')}
+        </p>
+        <hr className="my-4" />
+
+        <a className="btn btn-primary btn-lg" href={DONATE_GUIDESTAR_URL} role="button">
+          {I18n.t('about.donation_banner.donate_credit')}
+        </a>
```

Comment:
Should we use semantic-ui `Button` for these three?

Reply:
Yes please!

Reply:
Yep, that's fine